src/vision/status_quo/barbara_simulates_hydrodynamics.md

```diff
@@ -0,0 +1,75 @@
+# 😱 Status quo stories: Barbara Builds a Hydrodynamics Simulator
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today. 
+
+If you would like to expand on this story, or adjust the answers to the FAQ, feel free to open a PR making edits (but keep in mind that, as they reflect peoples' experiences, status quo stories [cannot be wrong], only inaccurate). Alternatively, you may wish to [add your own status quo story][htvsq]!
+
+## The story
+### Problem
+Barbara needed to build a tool to solve hydrodynamics simulations; there is a common method for this that subdivides a region into a grid and computes the solution for each grid patch. All the patches in a grid for a point in time are independent and can be computed in parallel, but they are dependent on neighboring patches in the previously computed frame in time.  This is a well known computational model and the patterns for basic parallelization are well established.
+
+Barabara wanted to write a performant tool to compute the solutions to the simulations of her research.  She chose Rust because she was already familiar with it and it had good qualities for writing performant code. After implementing the core mathematical formulas, Barbara began implementing the parallelization architecture. 
```

Comment:
```suggestion
Barabara wanted to write a performant tool to compute the solutions to the simulations of her research.  She chose Rust because she needed high performance but she also wanted something that could be maintained by her students, who are not professional programmers. Rust's safety guarantees give her confidence that her results are going to be corrupted from data races or other programming errors. After implementing the core mathematical formulas, Barbara began implementing the parallelization architecture. 
```

Reply:
Done